ui/page/send/layout.go

```diff
@@ -131,27 +101,23 @@ func (pg *Page) layoutDesktop(gtx layout.Context) layout.Dimensions {
 		layout.Stacked(func(gtx C) D {
 			gtx.Constraints.Min.Y = gtx.Constraints.Max.Y
 			return layout.S.Layout(gtx, func(gtx C) D {
-				return layout.Inset{Left: values.MarginPadding1}.Layout(gtx, pg.balanceSection)
+				return layout.Inset{Left: values.MarginPadding1}.Layout(gtx, wi.balanceSection)
 			})
 		}),
 	)
 
 	return dims
 }
 
-func (pg *Page) layoutMobile(gtx layout.Context) layout.Dimensions {
+func (wi *sharedProperties) layoutMobile(gtx C) D {
```

Comment:
Why the change from pg to wi? What I had in mind was keeping `pg` as is, adding a bool variable that you check when laying out the page.

Reply:
It would not work because page and modal are two different entity and are rendered differently.

Reply:
I can always change wi to pg.. the code is currently in 3 sections. The share layout codes, the send page and Modal. Both send and modal implement the share codes.

Reply:
What does `wi` stand for? I would get `sp` for sharedProperties.